test/e2e/kubectl/exec.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// OWNER = sig/cli
+
+package kubectl
+
+import (
+	"context"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	admissionapi "k8s.io/pod-security-admission/api"
+)
+
+var _ = SIGDescribe("Kubectl exec", func() {
+	f := framework.NewDefaultFramework("exec")
+
+	f.NamespacePodSecurityLevel = admissionapi.LevelBaseline
+	ginkgo.It("should be able to execute 5000 times in a container", f.WithSlow(), func(ctx context.Context) {
+		ns := f.Namespace.Name
+		podName := "test-exec-pod"
+		pod := e2epod.NewAgnhostPod(ns, podName, nil, nil, nil)
+		e2epod.NewPodClient(f).CreateSync(ctx, pod)
+		// run it 5000 times to check for leaks
+		errors := []error{}
+		for i := 0; i < 5000; i++ {
```

Comment:
how long does this test take?

Reply:
at least 5000s :) I moved it to a workers model so it executes 100 in parallel

Reply:
33 sec